### PR TITLE
fabla: init at 1.3.2

### PR DESCRIPTION
**Pull request by Horhik**
# Add Fabla. 
An open-source LV2 drum sampler plugin instrument

###### Motivation for this change
Needed more VST/LV2 plugins in nixpkgs

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
 - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/fabla/default.nix [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fabla";
+  version = "1.3.2";
+
+  src = fetchTarball {
+    url = "https://github.com/openAVproductions/openAV-Fabla/tarball/master";
+    sha256 = "1f6sh7y855xvhcnc9r73pdxnldwcs6f9ssvbypkwi5alkfa9zf14";
+  };
```
Comment (bobby285271) [inline pkgs/applications/audio/fabla/default.nix:26]: Please use `fetchFromGitHub` and track a specific tag.

File: pkgs/applications/audio/fabla/default.nix [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    libsndfile
+    libsamplerate
+    ntk
+    libXdmcp
+    libxcb
+  ];
+
+  meta = with lib; {
+    homepage = "http://openavproductions.com/fabla/"; 
+    description = "Fabla is an open-source LV2 drum sampler plugin instrument.";
```
Comment (bobby285271) [inline pkgs/applications/audio/fabla/default.nix:48]:
```suggestion
    description = "Fabla is an open-source LV2 drum sampler plugin instrument";
```

File: pkgs/applications/audio/fabla/default.nix [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ prusnak horhik];
```
Comment (bobby285271) [inline pkgs/applications/audio/fabla/default.nix:50]:
```suggestion
    maintainers = with maintainers; [ prusnak horhik ];
```

**Review by bobby285271 — COMMENTED** [review]
Please cleanup your git commit history.

**Comment by abathur** [comment]
@Horhik Are you still interested in trying to land this?

**Comment by SuperSandro2000** [comment]
This has been stale for over a year. I think it is fine to close it.

**Pull request closed without merging**